src/cata_utility.cpp

```diff
@@ -340,7 +340,6 @@ bool write_to_file_exclusive( const std::string &path,
 std::istream &safe_getline( std::istream &ins, std::string &str )
 {
     str.clear();
-    std::istream::sentry se( ins, true );
```

Comment:
This guy has strange side effects on the ins variable just because it is constructed, it needs to stay.